### ISSUE TITLE
Persistent configuration between sessions

### ISSUE THREAD

**Issue by micksmothers** [joeyberkovitz/plex-aaos#6]
_Note:_ enhancement request.

As a Plex-aaos user I want to have the settings and state of my playback persisted between sessions including:
* current playlist
* repeat status
* shuffle status
* current position in playlist
so that I can continue my listening where I left off without having to navigate the app to restore the configuration.

Additional bonus ask: perhaps persist the config per playlist so switching between playlists restores the state from a prior session with that playlist.

**Comment by joeyberkovitz** [comment]
Sounds good. My only concern with remembering settings per playlist is heavy disk usage which will only get worse once I start to support additional browsing methods (artist, ...). If that feature is important, maybe a buffer storing the settings for the last X playlists?

**Comment by micksmothers** [comment]
Yeah, I could see the storage issue with many playlists. Maybe a last x playlists as a settable option with a user selected range from 1 to 10 or something sane. Good idea, sir!
